frontend/src/components/Banner.test.tsx

```diff
@@ -35,6 +35,15 @@ describe('Banner', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('uses info mode when instructed', () => {
+    const tree = shallow(<Banner message={'Some message'} mode={'info'} />);
+    expect(tree.find('.icon').at(0)).toMatchInlineSnapshot(`
```

Comment:
nit: maybe for info mode, you can match the full snapshot, because other modes all got a full snapshot?